future-apps/payroll/test/helpers/setup.js

```diff
@@ -0,0 +1,78 @@
+module.exports = (artifacts, web3) => {
+  const { bigExp } = require('./numbers')(web3)
+  const { getEventArgument } = require('./events')
+  const getContract = name => artifacts.require(name)
+
+  async function deployErc20TokenAndDeposit(sender, finance, vault, name = 'ERC20Token', decimals = 18) {
+    const token = await getContract('MiniMeToken').new('0x0', '0x0', 0, name, decimals, 'E20', true) // dummy parameters for minime
+    const amount = bigExp(1e18, decimals)
+    await token.generateTokens(sender, amount)
+    await token.approve(finance.address, amount, { from: sender })
+    await finance.deposit(token.address, amount, 'Initial deployment', { from: sender })
+    return token
+  }
+
+  async function redistributeEth(finance) {
+    const split = 4
+    const amount = 10
+    const accounts = web3.eth.accounts
+
+    // transfer ETH to owner
+    for (let i = 1; i < split; i++)
+      await web3.eth.sendTransaction({ from: accounts[i], to: accounts[0], value: web3.toWei(amount, 'ether') })
```

Comment:
Long time ago, it was to then fund the payroll app from that account, I think, and then have enough funds for all payments. But I guess this can be avoided, for instance lowering salaries in some orders of magnitude if needed.